src/core/main.c

```diff
@@ -1699,6 +1699,17 @@ static void cmdline_take_random_seed(void) {
                    "This functionality should not be used outside of testing environments.");
 }
 
+static void set_suid_dumpable(void) {
+        const char *sd_path = "/proc/sys/fs/suid_dumpable";
```

Comment:
Uh, we already do this? https://github.com/systemd/systemd/blob/1b47cfab7f34c7c4a49d3320080398c206bc7f82/sysctl.d/50-coredump.conf.in#L37